source/onboard/ssl-client-certificate.rst

```diff
@@ -11,7 +11,8 @@ Before you begin, follow the `official guides to install Mattermost </guides/adm
 For the purposes of this guide, the Mattermost server domain name is ``example.mattermost.com``, and the user account is ``mmuser`` with email ``mmuser@mattermost.com`` and password ``mmuser-password``.
 
 .. note::
-  Generating the client certificates in this section is optional if you have already generated them before.
+  - Generating the client certificates in this section is optional if you have already generated them before.
+  - An SSL certificate (or a self-signed certificate) is required in cases where your Mattermost Server **Site URL** is configured as ``HTTP`` or ``localhost``.
```

Comment:
thanks @cwarnermm for enhancing the documentation around this issue 👍 . The `share-links.rst` seems like a good place to mention the potential problems with the copy link function, but I wonder if is there another place in the documentation that guides on how to invite members since that is also a very important place where the "copy link" function won't work without ssl certificate and can be annoying for users.
For the `ssl-client-certificate.rst`, maybe we should add like a strong suggestion on why it is really important to use it in a production environment (Security and Privacy, Compliance and Browser limitations) and mentioning that some product functionalities could be degraded because the lack of it, like in this case, the copy to clipboard functionality.

Reply:
@pvev - I've incorporated your feedback. Thank you again! Are there any other areas of product functionality that would be impacted that we'll want to call out via docs? Or would sharing links via messages and inviting people via URL be the only two?